.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I see what you mean. Also I realized that .vscode/settings is already checked in - let's keep this guy here :)